openssl-sys/build/main.rs

```diff
@@ -79,6 +79,11 @@ fn main() {
     let target = env::var("TARGET").unwrap();
 
     let (lib_dirs, include_dir) = find_openssl(&target);
+    let mut ossl_include_dir = include_dir.clone();
+    ossl_include_dir.push("openssl");
```

Comment:
This can just be `let ossl_include_dir = include_dir.join("openssl");` as far as I can tell